web_external/ImagesGallery/Detail/imageDetailsPage.jade

```diff
@@ -1,14 +1,33 @@
 .isic-image-details-container
-  .button
-    img(src='#{imgRoot}/ex.svg')
 
-  if image.id
-    .isic-image-details-name= image.name()
-      button.btn.btn-xs.btn-default.padleft.padbottom.zoom(data-toggle="tooltip" data-placement="bottom" title="View this image in full resolution")
+  .isic-image-details-heading
+    if image.id
+      span.isic-image-details-name= image.name()
+      button#isic-image-details-zoom.isic-image-details-button.btn.btn-xs.btn-default(
+        data-toggle='tooltip'
+        data-placement='bottom'
+        title='View this image in full resolution'
+      )
         i.icon-zoom-in
-      button.btn.btn-xs.btn-default.padleft.padbottom.openwindow(data-toggle="tooltip" data-placement="bottom" title="Open this image in a new window")
+      a#isic-image-details-openwindow.isic-image-details-button.btn.btn-xs.btn-default(
+        data-toggle='tooltip'
+        data-placement='bottom'
+        title='Open this image in a new window'
+        href='/api/v1/image/' + image.id + '/download?contentDisposition=inline'
+        target='_blank'
+      )
         i.icon-link-ext
+      a#isic-image-details-download.isic-image-details-button.btn.btn-xs.btn-default(
+        data-toggle='tooltip'
+        data-placement='bottom'
+        title='Download this image'
+        href='/api/v1/image/' + image.id + '/download'
```

Comment:
FYI I see a warning in Chrome:
```
Resource interpreted as Document but transferred with MIME type image/jpeg: "http://isic-archive.upstream:8080/api/v1/image/589de960d831136be37e0d8e/download".
```

This can be avoided by adding a `download` attribute to the anchor (described at https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a).

Also, regardless of the presence of that attribute I noticed that Firefox behaves differently in that it always presents a file save dialog. Chrome downloads the file immediately.